src/components/ImageFallback.js

```diff
@@ -13,7 +13,17 @@ const ImageFallback = ({ src, fallbackSrc, innerRef, ...props }) => {
     setFallback(false);
   }, [src]);
 
-  return <img src={fallback ? fallbackSrc : src} onError={handleError} ref={innerRef} {...props} />;
+  return (
+    <>
+      {props.cardid ? (
+        <a href={'/tool/card/' + props.cardid}>
+          <img src={fallback ? fallbackSrc : src} onError={handleError} ref={innerRef} {...props} />
+        </a>
+      ) : (
+        <img src={fallback ? fallbackSrc : src} onError={handleError} ref={innerRef} {...props} />
+      )}
+    </>
+  );
```

Comment:
This is supposed to be a generic component that is not necessarily specialized to card images. Can you add this to the `CardImage` component instead? Sorry to be a pain but i want to keep the basic components clean if possible